crates/typst-svg/src/lib.rs

```diff
@@ -452,6 +452,14 @@ impl SVGRenderer {
             Size::new(Abs::pt(width), Abs::pt(height)),
             self.text_paint_transform(state, &text.fill),
         );
+        if let Some(stroke) = &text.stroke {
+            self.write_stroke(
+                stroke,
+                Size::new(Abs::pt(width), Abs::pt(height)),
+                self.text_paint_transform(state, &text.fill),
+            );
+            self.xml.write_attribute("paint-order", "stroke"); // make sure the stroke is rendered on top of the fill
```

Comment:
Actually, it seems that we're already using `paint-order`. So Chrome is probably rendering "wrongly" because it's a very recent addition, so maybe it's just a matter of using a recent Chrome version (see https://caniuse.com/mdn-svg_attributes_presentation_paint-order )

Reply:
i dont know. i'm using arch and i think my chrome is relatively new. 

the image you posted seems only mentioned the order between stroke and fill but not the paint order of each glyphs. the stroke in that image is also not the "huge". i guess there is no one-key-solution for this problem.

for me i think such a huge stroke sounds like an edge case for me and i prefer not handling it in this pr. but i'm open for people who have better ideas.

Reply:
> the image you posted seems only mentioned the order between stroke and fill but not the paint order of each glyphs.
Hmmm... Good point. My thoughts regarding this parameter are, then, moot.

We'd need to find a way to draw the stroke for a group of glyphs at a time (e.g. a word). I'm not sure how feasible that is...

Maybe it's better to just leave it to another PR, honestly.
Still, thanks for the effort, the current additions are already awesome :slightly_smiling_face: